enumeratum-core/src/main/scala/enumeratum/EnumEntry.scala

```diff
@@ -31,8 +31,8 @@ object EnumEntry {
    *
    * http://stackoverflow.com/a/19832063/1814775
    */
-  private val regexp1: Pattern    = Pattern.compile("([A-Z]+)([A-Z][a-z])")
-  private val regexp2: Pattern    = Pattern.compile("([a-z\\d])([A-Z])")
+  private val regexp1: Pattern = Pattern.compile("([A-Z]+)([A-Z][a-z])")
```

Comment:
Since we don't need the class you added, please revert these formatting changes.

Reply:
Fixed.